azure-pipelines-master.yml

```diff
@@ -0,0 +1,28 @@
+# File: parking-permits/azure-pipelines-master.yml
```

Comment:
Hi Antti! thanks for the PR.

Could you please rename this file (and this comment line) to `azure-pipelines-develop.yml` for it to be consistent with the branch name it deploys from?

Reply:
Thanks Sam for review this is perfectly fine, we can rename it.

Reply:
Great, thanks!